build/status/MV9mZXRjaC9vdXQvcHJlY2lwX3NwYXRpYWwucmRzLmluZA.yml

```diff
@@ -1,10 +1,10 @@
 version: 0.3.0
 name: 1_fetch/out/precip_spatial.rds.ind
 type: file
-hash: 81d8d873ea5a0bda618a44d28d17ef14
-time: 2018-07-10 18:42:17 UTC
+hash: 975e7c74caa84925b60265d9122e71f0
+time: 2018-08-17 13:48:06 UTC
 depends:
-  view_polygon: 7ac3d28632d1dfbad8fd1b6044d4485a
+  view_polygon: 54debd6d6e335bbcdf505ce6bf1185f3
```

Comment:
cool, this looks like the same hash my system was rebuilding from. maybe it's not an OS issue after all. https://github.com/USGS-VIZLAB/vizstorm-GIF/commit/61eb5663b1243ada0123ae9b7a3d988b956b6a9a#diff-ec83309068922eefcf9d9b1d88fdd6ebR8